gensim/similarities/fastss.c

```diff
@@ -0,0 +1,8416 @@
+/* Generated by Cython 0.29.23 */
```

Comment:
This file seems like it's an auto-generated file which isn't related to this PR. If it is unrelated, then this file should be removed. If this file is related and required for this PR, then the reasons why should be explained in the description of this PR.